pkg/dns/coredns/pb/dns.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017 Istio Authors
```

Comment:
Copyright year should be 2020 in newer files

Reply:
On the first - no, we want to enable the CA functionality even if JWT tokens are not present. When we fixed VM support we added auth using previous cert ( or provisioning cert ), so JWT 
is no longer required for VM-only cases, nor will it be available. Part of the DNS changes we 
also want to improve the VM support.

Fixed the copyright - the code in ecosystem seems to be quite old...